tests/rgb/unit/stl.rs

```diff
@@ -7,35 +7,35 @@ use rgbstd::{
 
 #[tokio::test]
 async fn check_rgb20_stl() -> Result<()> {
-    let schema = rgb20_stl();
-    assert_eq!(LIB_ID_RGB20, schema.id().to_string());
+    let shema = rgb20_stl();
+    assert_eq!(LIB_ID_RGB20, shema.id().to_string());
```

Comment:
Why is this being put back? Was schema not misspelled?

Reply:
Merge problem, probably.

i will fix it, ok?

Reply:
Okay!